### PR TITLE
Create makefile

### PR DESCRIPTION
**Pull request by tswayd**
Hi Professor Zimmerman, Michali, and/or Kory! I finished the refactoring and makefile making homework assignment. I refactored the code in the process_sequences.sh by creating another if-then statement that's based on the second argument being "ALL". After that everything else seems to be working in place. Thank you for the informative lecture. I was overwhelmed at first but I slowly got it all to work eventually (I think). Let me know if you see any problems. Please and thank you. @naupaka @mhaliz @melt6457

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+	bash code/process_sequences.sh "$(SEQ_DATA)" > "$(SEQ_SUMMARY)"
+
+clean: 
+	rm -fv output/*.html
+	rm -fv Analysis.html
+	rm -fv output/*summary.txt
+	rm -fv output/*/*.csv
+	rm -fv output/*/*.png
+	rm -fv output/*/*/*.png
+	rmdir -fv output/Analysis_$state
```
Comment (melt6457) [inline Makefile]: When I run `make clean` on your project I get an error saying invalid option -- 'f'. I think you can safely remove this line. The `rm -fv output/*.html` on line 21 should take care of this

**Review by melt6457 — CHANGES_REQUESTED** [review]
@tswayd only one small change on this. Otherwise, nice work!! Its really cool to see your progress in this class over the semester.

File: Makefile (hunk 1 above)
Comment (tswayd) [reply Makefile]: Hi Kory thanks for your words! Did you mean line 27 `rmdir -fv output/Analysis_$state`? In which case I did remove that line. Let me know if you mean something else thank you.

**Comment by tswayd** [comment]
Hi Kory @melt6457 thanks for your words! Did you mean line 27 rmdir -fv output/Analysis_$state? In which case I did remove that line. Let me know if you mean something else thank you.

File: Makefile (hunk 1 above)
Comment (melt6457) [reply Makefile]: Yes, you got it!! Nice job Delong :)

**Review by melt6457 — APPROVED** [review]

**Comment by melt6457** [comment]
@tswayd great work on this! When you are ready merge this branch into your master branch. Then checkout your master branch on the server using `git checkout master` and then pull down your changes using `git pull origin master`

**Comment by tswayd** [comment]
> @tswayd great work on this! When you are ready merge this branch into your master branch. Then checkout your master branch on the server using `git checkout master` and then pull down your changes using `git pull origin master`

Thanks Kory I did it in the ssh command line and it seems to be up to date. Thank you very much for your help!

**Pull request merged**